source/blog/2019-01-04-the-ember-times-issue-79.md

```diff
@@ -1,39 +1,24 @@
 ---
 title: The Ember Times - Issue No. 79
-author: Chris Ng, Jessica Jordan, Kenneth Larsen, Michael Schinis, the crowd
+author: Chris Ng, Jessica Jordan, Kenneth Larsen, Michael Schinis
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/01/04-the-ember-times-issue-79.html"
 responsive: true
 ---
 
 Blein Vie Noa, Emberistas! 🐹🎆🎇
 
-The new year starts off with loads of **new RFCs**! Read more about **extending** the **Ember Inspector** 👓🐹, and suggested **deprecations** of **Route render** methods and selected **ApplicationController properties** �! We also have a new RFC for a brand new look of emberjs.com! And a new beta release of **ember-cli-babel**!
-
----
-
-## [SECTION TITLE](#section-url)
-
+The new year starts off with loads of **new RFCs**! Read more about **extending** the **Ember Inspector** 👓🐹, and suggested **deprecations** of **Route render** methods and selected **ApplicationController properties** �! We also have a new RFC for a brand new look of emberjs.com, **performance improvements** 🏎 for the API Docs **search**, a new beta release of **ember-cli-babel** and an advanced testing **exam** for you!
 
 ---
 
 ## [Website Redesign 🎨](https://github.com/emberjs/rfcs/pull/425)
-The official Ember website is due for a new look and feel. That is why a [new RFC](https://github.com/wifelette/rfcs/blob/master/text/0425-website-redesign.md) proposes a **completely new look** 💅 for emberjs.com. A side effect of not having updated the look and feel of the website for some years is that for people who aren’t involved in the day-to-day of Ember development, it’s **easy to miss just how well the framework has kept up over the years**. 
+The official Ember website is due for a new look and feel. That is why a [new RFC](https://github.com/wifelette/rfcs/blob/master/text/0425-website-redesign.md) proposes a **completely new look** 💅 for emberjs.com. A side effect of not having updated the look and feel of the website for some years is that for people who aren’t involved in the day-to-day of Ember development, it’s **easy to miss just how well the framework has kept up over the years**.
 
  The redesign aims to **modernize, update and improve all the things on the website** so that the impression given to the general public matches reality.
 
 There’s already a good discussion going on in the [RFC comment section](https://github.com/emberjs/rfcs/pull/425), and we would love to hear your thoughts as well.
 
----
-
-## [Ember Inspector (Extended)](https://github.com/emberjs/rfcs/pull/417)
```

Comment:
Was this an accidental delete?

Reply:
Unfortunately I wrote this section a couple of days ago and only realized today that the topic was already covered [in the previous times](https://github.com/emberjs/website/pull/3758/commits/360898be84cf5b09af3f6276518b54dc0d96b363) and therefore deleted it again

Reply:
Regarding this deletion then. I think it also needs to be deleted in the intro?